bucket/legit.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.2.0",
+    "description": "Complementary command-line interface for Git.",
+    "homepage": "https://frostming.github.io/legit/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://github.com/frostming/legit/blob/master/LICENSE"
+    },
+    "depends": "git",
+    "url": "https://github.com/frostming/legit/releases/download/1.2.0/legit.exe",
+    "hash": "2c4aa8a777141cad9171a7aec9b91d6570443644f7ca040903773620074d361b",
+    "post_install": "'y' | legit --install | Out-Null",
```

Comment:
```suggestion
    "installer": {
        "script": "'y' | legit --install | Out-Null"
    },
```

Reply:
Hm, this doesn't seem to work:
![image](https://user-images.githubusercontent.com/432127/81496988-da771b80-92bb-11ea-939d-73b7efa5d286.png)

Reply:
Interesting, it works for me:
![image](https://user-images.githubusercontent.com/16755704/81497051-47f86b80-92ff-11ea-9027-6f3cb0e07ef9.png)
Is there a better way to pass in that input?

Reply:
Do you use bash or pwsh/powershell? :smile:

Reply:
PowerShell. :)

Reply:
Okay, it works fine in Windows Sandbox. (PS 5 and 7)

![image](https://user-images.githubusercontent.com/432127/81497701-5bd0ad00-92c0-11ea-8a70-a584fea4daf0.png)